### PR TITLE
Change to support two TSCs

### PR DESCRIPTION
**Pull request by yanokwa**
The PMC has voted for the to-be-formed ODK 2 TSC to use the same goverance that the ODK 1 TSC is using.  

To that end, I've made a couple of changes to the document.
* Updated introduction to what we have on the ODK website (thus it's been approved)
* Replaced mentions of project with suite
* Added language to say we expect TSCs to have their own take on this document

I've also tweaked a bit of the grammar:
* Removed some unnecessary words
* Replaced smart quotes with dumb quotes 
* Replaced e.g. with e.g.

I've tried really hard to make sure we don't change the core meaning of the document and a review to confirm would be great.

As an aside, I toyed with replacing all "The TSC" with "Each TSC" or "Each suite's TSC" but it made for a pretty hard to parse. I think the document is pretty clear that starting from the Committer section, when we say "The TSC" we mean the suite's TSC.

### CODE REVIEW

File: TECHNICAL-STEERING-COMMITTEE.md [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
-* Suite roadmap (feature addition/removal, tool addition/removal,incorporating community feedback,etc.).
-* Forming appropriate Working Groups (e.g., User Feedback, Documentation, Translation) to gather the necessary community feedback before making decisions.
-* Technical resources (e.g., code repositories, servers)
-* Maintaining the list of Committers
- 
-Wherever possible, the TSCs should choose shared policies and infrastructure which allow for consistent process, but administrative separation across tool suites. In the case where the TSCs cannot reach consensus on policies and infrastructure, the final decision will be made by a majority vote by the PMC.
- 
-Although the TSC may update the TSC governance (e.g. this document) as it finds appropriate, revisions are subject to PMC review and veto, as ultimate authority over governing the ODK project rests with the PMC. Any update to the TSC governing document must be approved by the majority of the PMC. It is expected that the TSC will have a collaborative relationship (perhaps some overlap in membership) with the PMC.
```
[the remark this answers is not in the document]
Comment (yanokwa) [reply TECHNICAL-STEERING-COMMITTEE.md]: I'm going to push back here because I think adding it back makes the document harder to read. Or rather, when you read it with the old text, it's redundant since we've made it explict in a lot of places already.

At the very top, it says. 
>  The PMC is the ultimate authority over the project

Then in the relevant section, it says,
>  In the case where the TSCs cannot reach consensus...the final decision will be made by a majority vote by the PMC.

Then it goes on to say...
> Any update to the TSC governing document must be approved by the majority of the PMC.

I can't imagine the scenario where there is a dispute between the TSC where it isn't clear that the PMC is the decider. Is there a scenario that you have in mind?

[the remark this answers is not in the document]
Comment (wbrunette) [reply TECHNICAL-STEERING-COMMITTEE.md]: Fair points which is why I made it only a comment instead of a need changes, I was looking for discussion on it. 

My comment was this was a concern when we went through the draft/revision the first time and I wanted to make sure it got enough consideration this time. 

I am fine with it as you do point out it is covered in other areas.
